src/metrics/chaoss.ts

```diff
@@ -305,6 +305,66 @@ FORMAT JSONCompact`;
   });
 };
 
+interface IssueResponseTimeOptions {
+  type: 'avg' | 'median';
+  unit: 'week' | 'day' | 'hour' | 'minute';
+}
+export const chaossIssueResponseTime = async (config: QueryConfig<IssueResponseTimeOptions>) => {
+  config = getMergedConfig(config);
+  const whereClauses: string[] = ["type IN ('IssueCommentEvent','IssuesEvent') AND actor_login NOT LIKE '%[bot]'  "];
+  const repoWhereClause = await getRepoWhereClauseForClickhouse(config);
+  if (repoWhereClause) whereClauses.push(repoWhereClause);
+  const endDate = new Date(`${config.endYear}-${config.endMonth}-1`);
+  endDate.setMonth(config.endMonth);  // find next month  
+  let type = filterEnumType(config.options?.type, ['avg', 'median'], 'avg');
+  let unit = filterEnumType(config.options?.unit, ['week', 'day', 'hour', 'minute'], 'day');
+
+  const sql = `
+SELECT 
+ id,
+ argMax(name, time) AS name,   
+ ${getGroupArrayInsertAtClauseForClickhouse(config, { key: 'issue_response_time', defaultValue: 'NaN' })}
+FROM
+(
+  SELECT
+   ${getGroupTimeAndIdClauseForClickhouse(config, 'repo', 'issue_created_at')},
+   round(${type}(dateDiff('${unit}',issue_created_at, responsed_at)),${config.precision}) AS issue_response_time
+  FROM
+  (
+    SELECT
+      repo_id,
+      argMax(repo_name, created_at) AS repo_name,
+      org_id,
+      argMax(org_login, created_at) AS org_login,
+      issue_number,
+      minIf(created_at, action = 'opened' and issue_comments = 0  ) AS issue_created_at,
+      minIf(created_at, action = 'created' and actor_id != issue_author_id ) AS responsed_at
```

Comment:
@frank-zsy we can use 
`minIf(created_at, action in ( 'created','closed') and actor_id != issue_author_id ) AS responsed_at` 
instead of  
`minIf(created_at, action = 'created' and actor_id != issue_author_id ) AS responsed_at` to include close event.

Reply:
If I understand this correctly, we decide to ignore issues that has no comments and closed by the author him/herself. Is that right?

Reply:
> If I understand this correctly, we decide to ignore issues that has no comments and closed by the author him/herself. Is that right?

I think so, we should ignore the issues closed by author without comments.